server/routes/api/messages.js

```diff
@@ -36,11 +65,35 @@ router.post("/", async (req, res, next) => {
       senderId,
       text,
       conversationId: conversation.id,
+      isRead,
     });
     res.json({ message, sender });
+    return;
   } catch (error) {
     next(error);
   }
 });
 
+router.put("/read", (req, res, next) => {
+  const { otherUserId, conversationId } = req.body;
```

Comment:
You may want to protect this route by checking if there is req.user first.

Reply:
Got it I will add the check